app/controllers/reports_controller.rb

```diff
@@ -2,8 +2,21 @@
 class ReportsController < ApplicationController
   def create
     @info_request = InfoRequest.find_by_url_title!(params[:request_id])
+    if params[:comment_id]
```

Comment:
Also, once _anything_ has been reported once, you can no longer report further problems because of the `#attention_requested` attribute. Open to options about what you think might be a good way around this…